gandi/cli/modules/iaas.py

```diff
@@ -228,6 +228,9 @@ def create(cls, datacenter, memory, cores, ip_version, bandwidth,
             vm_params['ip_version'] = ip_version
             vm_params['bandwidth'] = bandwidth
 
+        if script:
+            vm_params['script'] = open(script).read()
```

Comment:
You should use a:
```
with open(script) as fd:
    vm_params['script'] = fd.read()
```
This way we are sure that the file descriptor is properly closed.